.vscode/settings.json

```diff
@@ -4,5 +4,6 @@
     "--load-plugins",
     "pylint_flask,pylint_flask_sqlalchemy"
   ],
-  "python.linting.pylintEnabled": true
+  "python.linting.pylintEnabled": true,
```

Comment:
Update vs code setting on your computer level not on the project level. Not everyone has the same python path.